ffi/capi/dart/package/test/test.dart

```diff
@@ -0,0 +1,46 @@
+import 'package:icu/icu.dart';
+import 'package:path/path.dart' as path;
+import 'package:test/test.dart';
+
+void main() {
+  init(path.absolute('test/libicu_capi_cdylib.so'));
+
+  test('ICU4XFixedDecimal.toString', () {
```

Comment:
nit: in the long run we should add more tests. We do this for C++ but not the other backends, but I'd like to have that tradition for backends we really want to be production-ready

Reply:
Those tests should be in the Diplomat repo though.

Reply:
Also, the Dart wrapper package intl4x will be wired up to the conformance project.